_posts/2018-06-29-upgrade-rails-from-5-0-to-5-1.markdown

```diff
@@ -27,11 +27,14 @@ your [Ruby on Rails](http://rubyonrails.org/) application from [version 5.0](htt
 
 Like Rails 5.0, [Rails 5.1](https://weblog.rubyonrails.org/2017/4/27/Rails-5-1-final/) requires [Ruby 2.2.2](https://www.ruby-lang.org/en/news/2015/04/13/ruby-2-2-2-released/) or later.
 
+If you want to know more about the Ruby versions that you could use, check out our
+[Ruby & Rails Compatibility Table](https://www.fastruby.io/blog/ruby/rails/versions/compatibility-table.html).
+
 <h2 id="gems">2. Gems</h2>
 
 - Make sure the gems you use are compatible with Rails 5.1, you can check this
```

Comment:
@etagwerker actually, right now you can't check this with RailsBump, right? since it only covers Rails 4.2, 5.2 and 6.0

Reply:
I guess it will do that in the near future. Maybe I should add an alternative to use next_rails? After all that's another way to do it.

Reply:
that sounds good 👍